Nation/AFDL/WillpowerExtraction.cs

```diff
@@ -102,9 +117,7 @@ public void uni19(int quant = 1)
 
         if (Core.IsMember)
         {
-            while (!Bot.ShouldExit && !Core.CheckInventory("Receipt of Swindle", 6))
-                Nation.SwindleReturn();
-            Core.BuyItem("tercessuinotlim", 1951, "Unidentified 19");
+            Adv.BuyItem("tercessuinotlim", 1951, "Unidentified 19");
```

Comment:
since u made this a single line the {}'s arent needed now. can go straight if this else that

Reply:
done